src/develop/develop.h

```diff
@@ -326,6 +326,12 @@ typedef struct dt_develop_t
     GtkWidget *button; // yes, ugliness is the norm. what did you expect ?
   } iso_12646;
 
+  // late scaling down from full roi
+  struct
+  {
+    GtkWidget *button;
+  } late_scaling;
```

Comment:
Add a gboolean here instead of the GtkWidget which is TRUE or FALSE depending on the fact that the full image processing is activated. This will make it possible to enable this easily in specific dev struct (during export, snapshot, clone...).